requirements/base.in

```diff
@@ -1,6 +1,6 @@
 # Core requirements for using this application
 -c constraints.txt
 
-pymongo>=2.7.2,<4.0.0
```

Comment:
You can leave minimum constraints in here.  Not sure if there is a reason for them, but it doesn't hurt anything to ensure they are kept.

Reply:
It seems harmless to keep them, but is there any value in it? I feel like as time progresses, the chances of accidentally going back past a min version goes to zero—but I'm also still really new to the Python dependency management world.

I suppose anything in `base.in` is actually used transitively, though, unlike the other files... so arbitrary dependency graph changes could result in an accidental downgrade.

Reply:
Not sure if we've been cleaning out or leaving old minimums in general.  It probably won't hurt to remove, but not sure.